internal/trace/gc.go

```diff
@@ -0,0 +1,825 @@
+// Copyright 2017 The Go Authors. All rights reserved.
```

Comment:
I think github has a couple of magic headers that can be used to auto hide “generated” content. Do you mind adding one of those so the diff when we sync isn’t as huge to review as it is now?

Reply:
Done, looks like it's working :)

Docs: 
https://help.github.com/en/github/administering-a-repository/customizing-how-changed-files-appear-on-github

Reply:
lovely